### ISSUE TITLE
Text orientation

### ISSUE THREAD

**Issue by matteodefra** [py-pdf/PyPDF2#1071]
## Explanation

Hello, I have some PDF documents which contains normal text with portrait orientation and a lateral text on the page side which is oriented in landscape mode. 
How can I tell PyPDF2 to extract only text that is oriented in portrait mode as the page orientation and to ignore the landscape text?

I attach an image example
![Screenshot from 2022-07-07 12-00-54](https://user-images.githubusercontent.com/52886292/177747695-6d6fb57c-4cdb-4c56-b872-d0f1bc3f7459.png)
Basically what I want is to extract the series of "example" above and ignore the three "example" rotated by 90 degrees

Thank you in advance for your help

**Comment by MartinThoma** [comment]
Some more evidence that people want this feature: https://stackoverflow.com/q/52530293/562769

**Comment by pubpub-zz** [comment]
@MartinThoma, @MasterOdin , @mtd91429,
the current parameters are the following: 
**extract_text( self, Tj_sep: str = "", TJ_sep: str = "", space_width: float = 200.0) -> str**

Tj_sep and TJ_sep are no more used : I would propose to take opportunity of introducing the orientation to remove them:
**extract_text( self, orientations : Tuple[int] = (0,90,270,360), space_width: float = 200.0) -> str**

Your opinion ?

**Comment by pubpub-zz** [comment]
some examples of calls:
page.extract_text(0) => extract all text strings oriented up
page.extract_text((0,)) => extract all text strings oriented up (synonym)
page.extract_text((0,180)) => extract all text strings oriented up or down

**Comment by MartinThoma** [comment]
Thank you so much @pubpub-zz ! I didn't think that this was possible :astonished: 

@matteodefra We will have a release towmorrow with this change.

**Comment by matteodefra** [comment]
Thank you so much @pubpub-zz and @MartinThoma !

**Comment by Liin159** [comment]
@matteodefra eventually, does this solution work?
